index.html

```diff
@@ -1,47 +1,77 @@
         <h1 id="title" class="center">Pele</h1>
         <p class="font"><i>The greatest football player of all time</i></p>
       </div>
       <div id="img-div">
-        <img class="center" id="image" src="https://cdn.britannica.com/38/211838-050-3D3D2391/Pele-Brazilian-soccer-player-athlete-circa-1970s.jpg" alt="Pele">
+        <img
+          class="center"
+          id="image"
+          src="https://cdn.britannica.com/38/211838-050-3D3D2391/Pele-Brazilian-soccer-player-athlete-circa-1970s.jpg"
+          alt="Pele"
+        />
         <div id="img-caption" class="font">Pele in action</div>
       </div>
       <div id="tribute-info" class="center">
-    <h2 class="font">Here's a time-line of Pele's life:</h2>
-    <ul class="font" style="text-align: justify;">
-        <br>
-        <li><b>1940</b> - Edson Arantes do Nascimento was born on October 23, 1940, in Três Corações, Brazil.</li>
-        <br>
-        <li><b>1956</b> - Pele began playing for Santos Football Club at the age of 16.</li>
-        <br>
-        <li><b>1958</b> - Pele scored two goals in the World Cup final against Sweden, helping Brazil win their first World Cup.</li>
-        <br>
-        <li><b>1962</b> - Pele suffered an injury in the second match of the World Cup but recovered to score four goals in the tournament, helping Brazil win their second World Cup.</li>
-        <br>
-        <li><b>1970</b> - Pele led Brazil to their third World Cup victory, scoring four goals in the tournament, including one in the final against Italy.</li>
-        <br>
-        <li><b>1974</b> - Pele retired from international football.</li>
-        <br>
-        <li><b>1975</b> - Pele played his last match for Santos Football Club.</li>
-        <br>
-        <li><b>1999</b> - Pele was named FIFA's "Player of the Century".</li>
-        <br>
-        <li><b>2021</b> - Pele turned 80 years old.</li>
-        <br>
-        <li><a id="tribute-link" href="https://en.wikipedia.org/wiki/Pel%C3%A9" target="_blank">Learn more about Pele's life on Wikipedia</a></li>
-    </ul>
-</div>
-
+        <h2 class="font">Here's a time-line of Pele's life:</h2>
+        <ul class="font" style="text-align: justify">
+          <br />
+          <li>
+            <b>1940</b> - Edson Arantes do Nascimento was born on October 23,
+            1940, in Três Corações, Brazil.
+          </li>
+          <br />
+          <li>
+            <b>1956</b> - Pele began playing for Santos Football Club at the age
+            of 16.
+          </li>
+          <br />
+          <li>
+            <b>1958</b> - Pele scored two goals in the World Cup final against
+            Sweden, helping Brazil win their first World Cup.
+          </li>
+          <br />
+          <li>
+            <b>1962</b> - Pele suffered an injury in the second match of the
+            World Cup but recovered to score four goals in the tournament,
+            helping Brazil win their second World Cup.
+          </li>
```

Comment:
Kindly replace deprecated tags such as <b> and <i> with the proper tags, this will help your code in being compatible with HTML5 specifications.